modules/insights-operator-copying-archive.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/remote-health-reporting-from-restricted-network.adoc
+
+[id="insights-operator-copying-archive_{context}"]
+= Copying an Insights Operator archive
+
+You must create a copy of your Insights Operator data archive for upload to link:https://cloud.redhat.com[cloud.redhat.com].
+
+.Prerequisites
+
+* You are logged in to {product-title} as `cluster-admin`.
+
+.Procedure
+
+. Find the name of the currently running Insights Operator pod:
```

Comment:
Small rewording consideration that might be helpful for localization (it makes sense/works as is though):

> Find the name of the Insights Operator pod that is currently running: